dev/com.ibm.ws.jsf.2.2/bnd.bnd

```diff
@@ -132,12 +132,14 @@ Import-Package: \
   !javax.ejb, \
   !javax.persistence, \
   !org.apache.myfaces.ee6, \
+  !net.sf.cglib.proxy, \
```

Comment:
What is this for?

Reply:
I can try removing - but this exclusion was because I got an unresolvable import if I didn't have it in....

Reply:
My local build didn't have any problems, so I think it should be ok to remove it. If you get any errors again, please let me know.